app/controllers/usersController.js

```diff
@@ -1,43 +1,31 @@
 const { registerUser } = require('../services/users');
-const { encryptPassword } = require('../helpers/encryption');
-const { isPasswordAlphanumeric, isPasswordLenghtvalid, isEmailValid } = require('../validators/users');
+const { isUserValid } = require('../validators/users');
 const logger = require('../logger/index');
 
 const addUser = (req, res) => {
-  const { firstName, lastName, email } = req.body;
-  let { password } = req.body;
+  const user = req.body;
 
-  if (isPasswordLenghtvalid(password) === false) {
-    res.status(500).send({ error: 'Password too short!' });
-    logger.error('Password too short!');
+  if (!isUserValid(user)) {
```

Comment:
Better like this, so you can save a line. Algo I think you can log the user data.
```
if (!isUserValid(user)) {
    logger.error(`User is not valid: ${user.status}`);
    return res.status(500).send({ error: 'Password too short!' });
  }
```